Encountify.Android/CustomMapRenderer.cs

```diff
@@ -187,7 +193,10 @@ void OnMyLocationChange(object sender, GoogleMap.MyLocationChangeEventArgs e)
                 {
                     try //This place might raise an exception during debugging but doesn't "seem" to crash the app
                     {
-                        CurrentPinWindow.ShowInfoWindow();
+                        lock(CurrentPinWindow)
+                        {
+                            CurrentPinWindow.ShowInfoWindow();
+                        }
```

Comment:
For me, when I try to touch the pin, there is now no infoWindow and it thorws an exception

Reply:
It also raises exception for me on the phone (though not on emulator), so I will remove it.